Cargo.toml

```diff
@@ -17,17 +17,18 @@ logging = ["env_logger"]
 dev = ["clippy"]
 
 [dependencies]
-ansi_term = "^0.7"
 log = "^0.3"
+xdg = "^2.0"
 tar = "^0.4"
-flate2 = "^0.2"
 curl = "^0.2"
-rustc-serialize = "^0.3"
-docopt = "^0.6"
 time = "^0.1"
+flate2 = "^0.2"
+docopt = "^0.6"
 walkdir = "^0.1"
-env_logger = { version = "^0.3", optional = true }
+ansi_term = "^0.7"
+rustc-serialize = "^0.3"
 clippy = {version = "~0.0.37", optional = true}
+env_logger = { version = "^0.3", optional = true }
```

Comment:
Didn't even realise I did that haha, it's personal preference really. I'm not really sure how most Rustaceans organise their `Cargo.toml`.